css/custom1.css

```diff
@@ -24,3 +24,7 @@
 .embed-container.ratio4x3 {
   padding-bottom: 75%;
 }
+
+.site-footer nav a + a {
+	margin-left: 1rem;
+}
```

Comment:
One could additionally imagine something like `.site-footer nav { margin-block: 0.5rem; }`, but I wanted to keep CSS changes to a minimum.

Reply:
Not sure what this would do - I am not really a CSS expert?

Reply:
My apologies, I should have been more explicit: That just adds some vertical spacing between footer children.

The first selector targets immediate children that have a preceding sibling whereas the second option uses flexbox, which is arguably simpler and more modern.

The `0.5rem` value is kind of arbitrary, but it seemed to fit in well with the values in `cayman.css`. (It's also [my own default value](https://prepitaph.org/snippets/html-boilerplate/) for such things.)

Reply:
Thanks, that is very helpful! I have put together c0c2694c4aa90448a52875ee261a05b9750de04f to solve this.

Reply:
That's not what I had originally intended - my suggestion was missing `flex-direction: column` 🤦 - but I kinda like what you've done with it:

![site footer with three columns on large viewport](https://github.com/user-attachments/assets/6d11681d-5166-49da-ab34-72e9191d33d2)

![site footer with three columns on small viewport](https://github.com/user-attachments/assets/21c814ef-cae1-4de0-8c94-7290ba5163f2)

However, I would then remove `margin-left: 1rem`, as that was only intended for horizontal separation of the two legal links.

FWIW, this is what I had in mind originally:

```css
.site-footer {
  display: flex;
  flex-direction: column;
  gap: 0.5rem;
}

.site-footer nav a + a {
  margin-left: 1rem;
}
```

![site footer with vertical separation](https://github.com/user-attachments/assets/74e178d5-5434-4e69-b499-fb4292eb02e2)